README.md

```diff
@@ -344,4 +344,36 @@ You can also retrieve the generated attributes as an array, which can be helpful
 {{ form_row(form.password, { attr: stimulus_target('hello-controller', 'a-target').toArray() }) }}
 ```
 
+### stimulus_outlet
+
+The `stimulus_outlet()` Twig function can be used to render [Stimulus Outlets](https://stimulus.hotwired.dev/reference/outlets).
+
+For example:
+
+```twig
+<div {{ stimulus_outlet('controller', 'foo', '.selector') }}>Hello</div>
```

Comment:
I haven't worked with outlets yet (I've only read the docs). If I'm correct, an outlet would always be on the same element as a controller, correct? So something like:

```
<div {{ stimulus_controller('hello') }} {{ stimulus_outlet('hello', 'foo', '.selector') }}>Hello</div<
```

Is that correct? If so, we should try to "fit" its syntax into `stimulus_controller` somehow - it feels unnecessary to require 2x `stimulus_` on the same element. For example, perhaps:

```
<div {{ stimulus_controller('hello')|stimulus_outlet('foo', '.selector')>
```

WDYT? Probably it would only make sense to make `stimulus_outlet` a filter - never its own function that could be used standalone. Or perhaps you can think of a better syntax. When `stimulus_controller` was created, CSS classes & outlets didn't exist... so it's getting kind of busy :p. For example, maybe this is better?

```
<div {{ stimulus_controller('hello').addOutlet('foo', '.selector')>
```

Reply:
Thank you @weaverryan for your answer (I'm a big fan of SymfonyCasts) !
You're right about the doc, I should have mentioned that an outlet would always be on the same element as a controller.
IMHO, a filter would not be the best thing to do because it would only work when coupled with a controller.
So you're right again : let me try to create a new function in StimulusControllersDto !

Reply:
Tell me if I'm wrong, but isn't there a mistake in StimulusControllersDto, line 39 ?

`$this->values['data-'.$controllerName.'-'.$key.'-class'] = $class;`

I guess it should be `$this->classes['...`, right ?

Reply:
Woh, yea, totally! Can you please fix it? It actually doesn't break anything, but we shouldn't leave it that way :)